prow/plugins/plugins.go

```diff
@@ -665,6 +665,14 @@ The list of patch release managers for each release can be found [here](https://
 			c.RequireMatchingLabel[i].GracePeriod = "5s"
 		}
 	}
+
+	if c.Golint == nil {
```

Comment:
Does the `Golint` struct need to be a pointer type? I think having just `minimumConfidence` as a pointer is sufficient.

Reply:
Why do we need omitempty? We do not write yaml config files, only read.